### DISCUSSION TITLE
using this with wezterm

### DISCUSSION THREAD

**Discussion by bryant-the-coder** [gokcehan/lf#872] [Q&A] [answered]
how to use this with wezterm?

**Comment by DusanLesan** [comment, marked as answer]
I have never heard about wezterm before but after installing and running it, lf seems to work without any issues. Does lf work for you in other terminals?

**Reply by bryant-the-coder** [reply]
can u show how u set it up on wezterm?

**Reply by DusanLesan** [reply]
I did nothing special. I have just installed wezterm and typed lf in it. Do you get some error when you run `lf`?

**Reply by bryant-the-coder** [reply]
i am on windows btw

**Reply by DusanLesan** [reply]
I use arch btw lol. I have some win VM and lf works well in powershell and cmd. Does it work for you there?

**Comment by CatsDeservePets** [comment]
It works under `Wezterm`. Not sure why it should not.
